lib/SQLiteReporter.cpp

```diff
@@ -293,7 +293,9 @@ void mull::SQLiteReporter::reportResults(const std::unique_ptr<Result> &result,
 
   sqlite3_close(database);
 
-  outs() << "Results can be found at '" << databasePath << "'\n";
+  outs() << "Results can be found at '" << databasePath << "'\n"
+         << "Use the following command to generate HTML report:\n"
+         << "REPORT=" << databasePath << " " << "make build\n";
```

Comment:
Well, this thing doesn't work on my machine because I use another shell.
Also, `make build` is part of another system.
Please, let's not add it here.